.github/workflows/pg-migtests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-22.04
     services:
       postgres:
-        image: postgres:13
+        image: postgres:14
```

Comment:
I think we should not update this PG version for this case, can we  move these cases to `misc-migtests.yml` ?